cabby/data/metagraph/utils.py

```diff
@@ -126,38 +133,6 @@ def update_osm_map(osm_map: Map,
   print("In update_osm_map: %d nodes in graph after adding." % (
     osm_map.nx_graph.number_of_nodes()))
 
-
-def add_conceptual_nodes_and_edges(graph: nx.Graph,
-                                   poi_map: Dict[str, int],
-                                   wd_relations: pd.DataFrame):
-  for _, row in wd_relations.iterrows():
-    place_node_id = poi_map[row["place"]]
-    graph.add_edge(row["instanceLabel"], place_node_id)
-    graph.add_edge(place_node_id, row["instanceLabel"])
-
-def poi_to_readable_name(poi_row: pd.Series) -> str:
-  """Gets a human-readable name for a POI (place of interest).
-
-  Arguments:
-    poi_row: a row of data from a map_structure.Map.poi data frame.
-  Returns:
-    name: a readable name for use as a node ID in a graph.
-  """
-  name = DEFAULT_POI_READABLE_NAME
-  if isinstance(poi_row["name"], str):
-      name = poi_row["name"]
-  elif isinstance(poi_row["wikipedia"], str):
-      name = poi_row["wikipedia"][3:]
-  elif isinstance(poi_row["brand"], str):
-      name = poi_row["brand"]
-  elif isinstance(poi_row["shop"], str):
-      name = poi_row["shop"]
-  elif isinstance(poi_row["tourism"], str):
-      name = poi_row["tourism"]
-  elif isinstance(poi_row["amenity"], str):
-      name = poi_row["amenity"]
-  return "%s_%s" % (name, poi_row["osmid"])
-
 def convert_multidi_to_weighted_undir_graph(
   in_graph: nx.MultiDiGraph, agg_function: Any) -> nx.Graph:
```

Comment:
Looks like agg_function can go away now? (The code that used it was deleted.)

Reply:
Ah, that was a mistake. In a3fe5f9 I add that functionality back in by first collecting the edge weights and then agg-ing them (instead of directly adding each edge in the loop).